resources/definitions/dagoma_neva_magis.def.json

```diff
@@ -0,0 +1,81 @@
+{
+    "id": "dagoma_neva_magis",
+    "name": "Dagoma NEVA",
```

Comment:
This printer has the same name as the other, that's why it is appearing twice in the list.